helpers.go

```diff
@@ -319,3 +322,44 @@ func joinAsArray(cmds []string, emptyArray bool) string {
 	}
 	return str + "]"
 }
+
+// hasMinZypperVersion determines the images's zypper version and compares
+// it with the required version. If the version cannot be determined, it
+// returns false.
+func hasMinZypperVersion(image, version string) bool {
+	requiredVersion := strings.Split(version, ".")
+	requiredMajor, _ := strconv.Atoi(requiredVersion[0])
+	requiredMinor, _ := strconv.Atoi(requiredVersion[1])
+	requiredPatch, _ := strconv.Atoi(requiredVersion[2])
+
+	buf := bytes.NewBuffer([]byte{})
+	id, err := runCommandInContainer(image, []string{"zypper --version"}, true, buf)
+	if err != nil {
+		log.Println("Unable determine zypper version:", err)
+		return false
+	}
+	removeContainer(id)
+
+	re := regexp.MustCompile(`(\d+)\.(\d+)\.(\d+)`)
```

Comment:
> Maybe we can use a golang alternative like https://github.com/mcuadros/go-version ?

This code uses a different algorithm from the one used by `rpm`. I would not trust it.

> Another solution is to just execute zypper lp --severity a and check if the output is Unknown option '--severty'. A bit hackish, but it would solve this.

Yes, that would be way easier.